src/exercise-two.js

```diff
@@ -0,0 +1,13 @@
+// 2. Implement a JavaScript function to calculate the Fibonacci sequence up to a given number.
+
+const fibonacci = (fibonacci) => {
+  let array = [0, 1];
+  while (array.length < fibonacci) {
+    let sum = array.at(-1) + array.at(-2);
+    array.push(sum);
+    sum = 0;
+  }
```

Comment:
Some people would prefer a for loop here since it keeps all the loop control statements on one line which can be easy to follow and saves some lines of code
If you would like to keep your logic with a for loop I think this may work

```js
for (let sum = 0; array.length < fibonacci; sum = 0){
  sum = array.at(-1) + array.at(-2);
  array.push(sum);
}
```

Reply:
yeah this can work

Reply:
but last sum=0 is not neccesary